doc/migrations/6.2.1_x.x.x.md

```diff
@@ -107,4 +107,23 @@ Stop the cluster, or individual nodes, if performing a rolling upgrade, and exec
     3. Add Prometheus to the [instrumentation section](../configuration/instrumentation.md).
     4. Configure a [listener](../listeners/listen-http.md#handler-types-prometheus-mongoose_prometheus_handler) for Prometheus.
 
-Restart the node or cluster.
\ No newline at end of file
+Restart the node or cluster.
+
+## Support for the new RDBMS database - CockroachDB
+
+We've introduced support for a new RDBMS database, CockroachDB. To ensure compatibility and proper functionality, we've also added a new test preset: `cockroachdb_cets`.
```

Comment:
I am not sure if the preset info is of any value for the user, probably not... Maybe put it in the release notes instead?